fhir-validation/src/main/java/com/ibm/fhir/validation/FHIRValidator.java

```diff
@@ -37,21 +40,19 @@
 import com.ibm.fhir.validation.exception.FHIRValidationException;
 
 public class FHIRValidator {
-    public static boolean DEBUG = false;
-
-    private ValidatingNodeVisitor visitor = new ValidatingNodeVisitor();
+    private static final Logger log = Logger.getLogger(FHIRValidator.class.getName());
 
     private FHIRValidator() { }
 
     /**
      * Validate a {@link Resource} against constraints in the base specification and
      * resource-asserted profile references or specific profile references but not both.
      *
-     * <p>Resource-asserted profiles which are unknown to the server will be ignored. Conversely,
-     * unknown profiles passed explicitly as arguments will result in a FHIRValidationException.
+     * <p>Resource-asserted profile references that are unknown, generate an issue with severity WARNING. Profile references
+     * passed in as arguments to this method, generate an issue with severity ERROR.
```

Comment:
```suggestion
     * <p>Resource-asserted profile references that are unknown to the server generate an issue with severity WARNING.
     * <p>Unknown profile references passed as arguments to this method generate an issue with severity ERROR.
```

Reply:
and the same change in each of the other variants

Reply:
Updated Javadoc to reflect this language